WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -290,15 +281,17 @@ public static void showSignInForResult(Activity activity) {
         }
     }
 
-    public static void viewStatsSinglePostDetails(Context context, SiteModel site, Post post, boolean isPage) {
+    public static void viewStatsSinglePostDetails(Context context, SiteModel site, PostModel post, boolean isPage) {
         if (post == null) return;
 
-        PostModel postModel = new PostModel(site.getSiteId(), post.getRemotePostId(), post.getTitle(), post
-                .getLink(), isPage ? StatsConstants.ITEM_TYPE_PAGE : StatsConstants.ITEM_TYPE_POST);
+        org.wordpress.android.ui.stats.models.PostModel postModel =
```

Comment:
I had decided against this to avoid making it inconsistent with the rest of the stats models:

<img width="158" alt="stats-models" src="https://cloud.githubusercontent.com/assets/9613966/19384592/a695ab86-9212-11e6-9e87-c81beb3c2877.png">

But on reflection, you're probably right that this is worse (and anyway a lot of those models will be moved to FluxC at some point in the future). I'll change it.